macos/FluentUI/Button/Button.swift

```diff
@@ -139,6 +146,32 @@ open class Button: NSButton {
 			}
 		}
 	}
+	
+	/// the current state of the button
+	private var buttonState: ButtonState? {
```

Comment:
nit: move private variables to the bottom: https://microsoft.github.io/swift-guide/FileContentOrdering.html

Reply:
But also (this can be followup after this is merged) we could make this public, and then make `isPressed` private? I think @chripear made `isPressed` public just to make things easier to test in the test app, and this seems like the better place to do it

Reply:
@Saadnajmi thanks for pointing me to the guide, will reorder :)